core/src/epicli/tests/helpers/constants.py

```diff
@@ -0,0 +1,44 @@
+OUTPUT_PATH = '/workspaces/epiphany/core/src/epicli/test_results/'
+CLUSTER_NAME_SAVE = 'test-save'
+CLUSTER_NAME_LOAD = 'test-load'
+NON_EXISTING_CLUSTER = 'test-aaaa'
+TEST_DOCS = \
```

Comment:
No need for backslashes.

Reply:
Also pls space out all these structures properly to make them more readable. For examle:

```json
{
   "kind":"epiphany-cluster",
   "title":"Epiphany cluster Config",
   "provider":"azure",
   "name":"CLUSTER_NAME_SAVE",
   "specification":{
      "prefix":"test",
      "name":"CLUSTER_NAME_SAVE",
      "admin_user":{
         "name":"operations",
         "key_path":"id_rsa"
      },
      "cloud":{
         "subscription_name":"Test-Dev",
         "vnet_address_pool":"10.1.0.0/20",
         "use_public_ips":false,
         "use_service_principal":false,
         "region":"West Europe",
         "credentials":{
            "key":"1111-1111-1111",
            "secret":"XXXXXXXXXXXXXXX"
         },
         "default_os_image":"default"
      }
   }
}
```

Packing them together just makes it hard to read.